.travis.yml

```
@@ -29,7 +29,7 @@ addons:
       - libgoogle-perftools-dev
 
 env:
-  - BAZEL_VERSION=4.1.0 ENABLE_HEAPCHECK=1
+  - BAZEL_VERSION=4.1.0 ENABLE_HEAPCHECK=1 LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8 LANGUAGE=en_US.UTF-8
```

```suggestion
  - BAZEL_VERSION=4.1.0 ENABLE_HEAPCHECK=1
```

Should resolve error:

```bash
UnicodeEncodeError: 'ascii' codec can't encode characters in position 73-75: ordinal not in range(128)
```

---

Reverted. The build container is still running Python 3.6:

```bash
$ which python
/opt/pyenv/shims/python
The command "which python" exited with 0.
$ python -V
Python 2.7.17
The command "python -V" exited with 0.
$ which python3
/opt/pyenv/shims/python3
The command "which python3" exited with 0.
$ python3 -V
Python 3.6.9
The command "python3 -V" exited with 0.
```